modules/floating-bars/assets/scss/widgets/floating-bars-vars.scss

```diff
@@ -0,0 +1,3 @@
+$floating-bars-border-radius-round: 50px;
+$floating-bars-border-radius-rounded: 20px;
+$floating-bars-border-radius-sharp: 0;
```

Comment:
Why not implement SASS variables directly in the `modules/floating-bars/assets/scss/widgets/floating-bars.scss` file?

Reply:
Hi! Sorry this was just a draft PR, it's still WIP, will close and re-open when it's ready.
Anyway, to answer your question, there will be more variations of this widget that will use the same variables, this is why they are in this file.